localizations/fa.json

```diff
@@ -0,0 +1,65 @@
+{
+	"Language": "فارسی",
```

Comment:
this is amazing ;)

Reply:
I'll ask my friends to do more translations in their native language. :D